README.md

```diff
@@ -61,46 +67,114 @@ To remove `assume-unchanged` from all files with that currently configured:
 git ls-files -v|grep '^h'| sed 's/^h//' | xargs git update-index --no-assume-unchanged
 ```
 
-### Regenerating data files
+## Data generation
 
-#### Using Travis CI
+The static-budget-portal is the front-end (static site) of Vulekamali and does not generate or retrieve data.
 
-Every branch push results in a Travis CI build, but by default it won't regenerate data files.
+The static-budget-portal requests data from the [Data Manager](https://github.com/OpenUpSA/budget-portal) 
+and stores it locally in the `_data` directory, which it then uses to build the static site via Jekyll.
 
-Add `[ci]` to the commit message of the latest commit in the pushed branch to opt into generating data files.
+That means the actual data being served in production is managed by Git.
```

Comment:
could say "stored in the git repository alongside the code" instead of "managed by git"